labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,10 +32,46 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
-  def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+  
+  // Not worth it
+  
+    def googleCodeJamGooglerese(lines: String*): Seq[String] = {
+    if (lines.length == 0) return Seq("")
+    var letter: Char = lines.head(0)
+    var outLetter: Char = ' ' 
+//    letter match {
+//      case 'a' => outLetter = 'y'
+//      case 'b' => outLetter = ' '
+//      case 'c' => outLetter = 'e'
+//      case 'd' => outLetter = 's'
+//      case 'e' => outLetter = 'o'
+//      case 'f' => outLetter = 'c'
+//      case 'g' => outLetter = 'v'
+//      case 'h' => outLetter = ' '
+//      case 'i' => outLetter = ' '
+//      case 'j' => outLetter = 'u'
+//      case 'k' => outLetter = 'i'
+//      case 'l' => outLetter = 'g'
+//      case 'm' => outLetter = 'l'
+//      case 'n' => outLetter = 'b'
+//      case 'o' => outLetter = 'k'
+//      case 'p' => outLetter = 'r'
+//      case 'q' => outLetter = 'z'
+//      case 'r' => outLetter = 't'
+//      case 's' => outLetter = 'n'
+//      case 't' => outLetter = 'w'
+//      case 'u' => outLetter = ' '
+//      case 'v' => outLetter = 'p'
+//      case 'w' => outLetter = 'f'
+//      case 'x' => outLetter = 'm'
+//      case 'y' => outLetter = 'a'
+//      case 'z' => outLetter = ' '
+//      case ' ' => outLetter = ' '
+//    }
+    return Seq(outLetter.toString) ++ googleCodeJamGooglerese(lines.drop(1).toString)
```

Comment:
I think there might be a way to build the mapping programmatically from the examples.